sequence.go

```diff
@@ -0,0 +1,84 @@
+package gween
+
+type (
+	// Sequence represents a sequence of Tweens, executed one after the other.
+	Sequence struct {
+		Tweens []*Tween
+		index  int
+	}
+)
+
+// NewSequence returns a new Sequence object.
+func NewSequence() *Sequence {
+	seq := &Sequence{
+		Tweens: []*Tween{},
+	}
+	return seq
+}
+
+// Add adds one or more tweens in order to the Sequence.
+func (seq *Sequence) Add(tweens ...*Tween) {
+	for _, tween := range tweens {
+		seq.Tweens = append(seq.Tweens, tween)
+	}
+}
+
+// Remove removes one or more specified tweens from the Sequence.
+func (seq *Sequence) Remove(tweens ...*Tween) {
+	for _, tween := range tweens {
+		for i, t := range seq.Tweens {
+			if t == tween {
+				seq.Tweens = append(seq.Tweens[:i], seq.Tweens[i+1:]...)
```

Comment:
This will not always work because you are altering the slice as you iterate it. I recommend making a private method that removes a single tween and then iterating the passed in tweens to remove one at a time.

Reply:
Actually, this _should_ always work, if I'm correct? It's modifying the Tweens array once for each tween to remove; after that, it breaks out of the for loop, so it never runs into an issue of the iterator going out of bounds or skipping tweens.